<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>770</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>770</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>770</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;
color: #37474f;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainArea">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>1100</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1100</width>
       <height>700</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane
{
	top: -34px; 
	border-width: 2px;
	border-color: #37474f;
	border-style: solid;
}

QTabBar::tab
{
	
	background: #80c683;
	border-width: 2px;
	border-style: solid;
	border-color: #37474f;
 	height: 30px;
	width: 300px;
	color: #37474f;
}

QTabBar::tab:selected
{
	
	background:#80c683;
	color: #F4F4EF;
}

QTabBar::tab:hover
{
	background:#80c683;
	color: #F4F4EF;
}

QTabBar::tab:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #80c683, stop: 1 #4fc3f7);
}
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>BEFORE</string>
      </attribute>
      <widget class="MyGraphicsView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>900</width>
         <height>600</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
      <widget class="QLabel" name="fileName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>670</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #37474f;</string>
       </property>
       <property name="text">
        <string>File Name</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>AFTER</string>
      </attribute>
      <widget class="MyGraphicsView" name="graphicsView_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>900</width>
         <height>600</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
      <widget class="QLabel" name="fileName_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>670</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #37474f;</string>
       </property>
       <property name="text">
        <string>File Name</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="leftMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>200</width>
      <height>700</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="openFile">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>140</y>
       <width>65</width>
       <height>65</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icons/img/download (2).png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="saveFile">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>210</y>
       <width>65</width>
       <height>65</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icons/img/upload (1).png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelChanges">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>350</y>
       <width>65</width>
       <height>65</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icons/img/synchronize.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="filters">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>280</y>
       <width>65</width>
       <height>65</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icons/img/edit.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="emboss">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>EMBOSS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sharpen">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>175</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>SHARPEN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="box_blur">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>BOX BLUR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gaussian">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>245</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>GAUSSIAN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="laplacian">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>LAPLACIAN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sobelLeft">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>315</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>SOBEL LEFT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="user_kernel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>385</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>USER KERNEL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sobelRight">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>SOBEL RIGHT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="discolor">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>455</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>DISCOLOR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="negative">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>490</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>NEGATIVE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toning">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>525</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>TONING</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>595</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>HSV FILTERS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>630</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>HSL FILTERS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>665</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
text-align: left;</string>
     </property>
     <property name="text">
      <string>BRIGHTNESS /
CONTRAST</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>560</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #37474f;
</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="menu">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>15</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	image: url(:/icons/img/menu.png);
}





</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>15</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/img/cancel.png);
image: url(:/icons/img/failed.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>leftMenu</zorder>
   <zorder>menu</zorder>
   <zorder>mainArea</zorder>
   <zorder>close</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Open_File"/>
    <addaction name="Save_File"/>
    <addaction name="Close_program"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="Brightness"/>
    <addaction name="Contrast"/>
    <addaction name="Saturation"/>
    <addaction name="separator"/>
    <addaction name="Desaturation"/>
    <addaction name="Negative"/>
    <addaction name="Toning"/>
   </widget>
   <widget class="QMenu" name="menuFilter_2">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="Emboss"/>
    <addaction name="Sharpen"/>
    <addaction name="Box_blur"/>
    <addaction name="Gaussian_blur"/>
    <addaction name="Laplacian_filter"/>
    <addaction name="Prewitt_filter"/>
    <addaction name="separator"/>
    <addaction name="Arbitary_filter_kernel_matrix"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
   <addaction name="menuFilter_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Open_File">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/img/download.png</normaloff>:/icons/img/download.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="Save_File">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/img/upload.png</normaloff>:/icons/img/upload.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="Close_program">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/img/delete-folder.png</normaloff>:/icons/img/delete-folder.png</iconset>
   </property>
   <property name="text">
    <string>Close program</string>
   </property>
  </action>
  <action name="Desaturation">
   <property name="text">
    <string>Desaturation</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="Negative">
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="Toning">
   <property name="text">
    <string>Toning</string>
   </property>
  </action>
  <action name="Brightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="Contrast">
   <property name="text">
    <string>Contrast</string>
   </property>
  </action>
  <action name="Saturation">
   <property name="text">
    <string>Sturation</string>
   </property>
   <property name="toolTip">
    <string>Saturation</string>
   </property>
  </action>
  <action name="Emboss">
   <property name="text">
    <string>Emboss</string>
   </property>
  </action>
  <action name="Sharpen">
   <property name="text">
    <string>Sharpen</string>
   </property>
  </action>
  <action name="Box_blur">
   <property name="text">
    <string>Box blur</string>
   </property>
  </action>
  <action name="Gaussian_blur">
   <property name="text">
    <string>Gaussian blur</string>
   </property>
  </action>
  <action name="Laplacian_filter">
   <property name="text">
    <string>Laplacian filter</string>
   </property>
  </action>
  <action name="Prewitt_filter">
   <property name="text">
    <string>Prewitt filter</string>
   </property>
  </action>
  <action name="Arbitary_filter_kernel_matrix">
   <property name="text">
    <string>Arbitary filter kernel matrix</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
